_projects/vaccines-dot-gov.md

```diff
@@ -0,0 +1,70 @@
+
+<blockquote class="pullquote" markdown="1">
+We examined the product under a microscope, thinking about everything USDS knows about product launches and checking for things like usability and stability.
+ <footer>– Project Lead, Hannah Pyper</footer>
+</blockquote>
+
+USDS and our agency partners worked to prepare the site for a wide range of users with the following updates:
+- Made sure all language was inclusive and plain
+- Complied code and design with the federal government’s accessibility standard for people with disabilities called Section 508. This also made the site easier to use for everyone, regardless of physical abilities.
+- Improved the site to go faster and perform better on mobile devices
+
+It became clear from user research that one website wouldn’t meet the needs of the public and we needed ways to reach people without Internet.
+Working with public and private sector partners, the team created a Spanish-language website, Vacunas.gov, and WhatsApp chatbot to reach Spanish speakers. A call center and an SMS short code tool, in English, Spanish, and additional languages, were also launched to increase reach.  
+
+
+![](../images/project-vaccines-chatbot-screenshot.jpg)
+
+## The Impact
+
+How do you prepare for an onslaught of web traffic right away? Suspecting that President Biden might mention the website in a speech on the anniversary of the COVID-19 pandemic, systems engineers readied the site to host 100 million users simultaneously. Traffic never climbed that high, but preparing for such a high number influenced the success of the launch. 
+
+We partnered with Google, Facebook, Univision, and others to launch identical vaccine finders on their platforms. This allowed vaccine availability data from pharmacies, states, and local jurisdictions provided to the CDC to reach an even wider swath of people, in the way that they wanted to search for this information. The Vaccines.gov data is also published on data.cdc.gov, available for anyone to find and use as they need to serve the public interest.
+
+We also made a public API for this project in partnership with pharmacies, states, local clinics, and electronic health records. This API allows anyone to access vaccine appointment availability data and use the data to build more digital projects. The Appointment Availability API was also the first FHIR (fast healthcare interoperability resources) API connecting pharmacy, state, and local jurisdiction data.
+
+
+![](../images/project-vaccines-univision-vaccine-finder-tool.jpg)
+*See a [video of the vaccine finder tool in action](https://www.univision.com/noticias/salud/donde-hay-vacunas-contra-el-covid-19-disponibles-en-mi-zona).*
+
+Parallels to Healthcare.gov were drawn as soon as the project was announced. Hsiang, who worked on both Healthcare.gov and Vaccines.gov, said the government has come a long way since 2014. Chiefly, technologists had a seat at the table from the beginning of Vaccines.gov, unlike Healthcare.gov.
```

Comment:
```suggestion
Parallels to Healthcare.gov were drawn as soon as the project was announced. Mina Hsiang, who worked on both Healthcare.gov and Vaccines.gov, said the government has come a long way since 2014. Chiefly, technologists had a seat at the table from the beginning of Vaccines.gov, unlike Healthcare.gov.
```